front/lib/api/assistant/global_agents.ts

```diff
@@ -245,8 +214,8 @@ function _getGPT4GlobalAgent({
     scope: "global",
     userListStatus: status === "active" ? "in-list" : "not-in-list",
     model: {
-      providerId: GPT_4O_MODEL_CONFIG.providerId,
-      modelId: GPT_4O_MODEL_CONFIG.modelId,
+      providerId: GPT_4O_MINI_MODEL_CONFIG.providerId,
```

Comment:
? We def don't want to point gpt4 to the mini model?

Reply:
definitely not, don't know what I did here